ch_pipeline/processing/daily.py

```diff
@@ -213,15 +213,14 @@
     # flagged regions
     - type: draco.analysis.flagging.ThresholdVisWeight
       in: sstream
-      out: sstream_threshold
+      out: weight_mask
       params:
-          relative_threshold: 0.5
```

Comment:
Is there any rationale to changing this threshold?

Reply:
With the change from the mean/median being per-baseline to per-frequency, the relative threshold can be more aggressive without cutting off all of the higher frequencies. A relative threshold of 0.5 let a lot of bad data through, although 0.9 might be a bit too aggressive. Depends on the behaviour that we want, but I figure its better to be safe than to let bad data throughy

Reply:
Ideally we'd have some tests where we see the effects on the data of tweaking that param. I think there's two competing effects in the changes that have been made:

- Median is now per frequency and so should be more representative of the actual level. That means upping the level makes sense.
- Median now excludes missing samples when it is being calculated. This was biasing the level calculation low, and so to maintain a similar level of excision we would need to *lower* the threshold, but we are tightening it.

Overall I'd guess that 0.9 is too high. The regridder is fairly successful when only a few samples are missing, but this would cut those regions out as they'd probably always lower the weights by 10%.

I guess maybe we should have a good look at the data to see what the effects are.